requirements.txt

```diff
@@ -0,0 +1,4 @@
+paramiko==3.5.1
+PyQt5==5.15.11
+PyQt5_sip==12.12.2
+simplejson==3.20.1
```

Comment:
I don't like pinning versions so specifically here, unless there's actually some version limitation.  This ends up breaking our env build more than it helps, particularly whenever we want to update something to get new functionaty

I think normally we also get the sip packages along with `PyQt5`, so we don't need to specify it here